addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -1880,6 +1880,8 @@ body.editor_enable.editor_has_snippets {
             .o_remove_color {
                 font-size: 14px !important;
                 text-align: center !important;
+                overflow: hidden;
```

Comment:
I am always suspicious of `overflow: hidden` rules because most of them are used when not needed and hiding bug instead of fixing them: could you explain why it is needed in this case? Thanks :+1:

Reply:
I thought it was necessary to fix the bug but it seems like just switching the padding to 0 fixes it. I'm a little confused as it's one of the first thing I tried but I guess my testing was flawed 🤔